style.css

```diff
@@ -3,7 +3,7 @@
 body {
   margin: 0;
   width: 375px;
-  height: 4769px;
+  height: 5410px;
```

Comment:
- [Optional] Please the design should match the figma template for all mobile sizes not just the 375px width design look how your design becomes after increasing the screen size a little bit.
![image](https://user-images.githubusercontent.com/74744787/214619229-9a29d41f-71eb-41b8-ae62-45f79e96011e.png)

Reply:
implemented